Content.Server/Shuttles/Systems/ShuttleSystem.FasterThanLight.cs

```diff
@@ -328,6 +328,14 @@ public void FTLToDock(
         // Valid dock for now time so just use that as the target.
         if (config != null)
         {
+            // Moffstation - Start - Valid dock so mark each one as queued
+            config.Docks.ForEach(i =>
+            {
+                i.DockA.Queued = true;
+                i.DockB.Queued = true;
+            });
```

Comment:
```suggestion
            foreach (var dock in config.Docks)
            {
                dock.DockA.Queued = true;
                dock.DockB.Queued = true;
            }
```
LINQ is generally considered to be so non-performant as to never be used unless it's absolutely necessary.